303-quarkus-vertx-sql/src/test/java/io/quarkus/qe/vertx/sql/handlers/airline/Db2AirlineHandlerTest.java

```diff
@@ -1,10 +1,10 @@
-package io.quarkus.qe.vertx.sql.handlers.Airline;
+package io.quarkus.qe.vertx.sql.handlers.airline;
 
 import io.quarkus.qe.vertx.sql.test.resources.Db2TestProfile;
 import io.quarkus.test.junit.QuarkusTest;
 import io.quarkus.test.junit.TestProfile;
 
 @QuarkusTest
 @TestProfile(Db2TestProfile.class)
-public class Db2HandlerTest extends AirlineHandlerTest {
+public class Db2AirlineHandlerTest extends AirlineHandlerTest {
```

Comment:
Redundant name -> Db2HandlerTest  applies to this context `io.quarkus.qe.vertx.sql.handlers.airline`

Reply:
this comment applies to all bellow changes.

Reply:
This is to have unique classes names. Before, we had a few Db2HandlerTest classes. Having unique names is useful when troubleshoting test failures or search by a class name.